cmd/agent_autopilot.go

```diff
@@ -151,10 +174,107 @@ var agentAutopilotCmd = &cobra.Command{
 	},
 }
 
+func paymentDue(frequency float64, chainHeadHeight, epochsPaid *big.Int) bool {
+	epochFreq := big.NewFloat(float64(frequency * constants.EpochsInDay))
+
+	epochsPassed := new(big.Int).Sub(chainHeadHeight, epochsPaid)
+
+	epochFreqInt64, _ := epochFreq.Int64()
+	epochFreqInt := big.NewInt(epochFreqInt64)
+
+	return epochsPassed.Cmp(epochFreqInt) >= 0
+}
+
+// needToPullFunds returns whether the payAmt is larger than the current
+// balance of the Operator wallet.
```

Comment:
I think this comment is just wrong ... it's checking the "Agent Liquid Assets". The operator wallet balance is just for gas fees.

Reply:
Updated.